_posts/2024-02-01-testing-with-wildfly-glow.adoc

```diff
@@ -0,0 +1,182 @@
+---
+layout: post
+title:  "WildFly testing with WildFly Glow"
+date:   2024-01-10
+tags: galleon provisioning wildfly-glow arquillian test wildfly-maven-plugin wildfly
+author: jdenise
+description: WildFly testing with WildFly Glow.
+---
+
+This blog post provides information on how to use https://github.com/wildfly/wildfly-glow[WildFly Glow] to produce a 
+WildFly server in order to test the applications you are developing for WildFly.
+
+Note: reading this http://XXXX[blog post] that provides detailed information on WildFly provisioning in general and WildFly Glow in particular would be 
```

Comment:
Use asciidoc [NOTE]

s/would be/is/g

remove 'prior' as 'prerequisite' means prior.